src/theory/strings/theory_strings.h

```diff
@@ -602,6 +608,21 @@ class TheoryStrings : public Theory {
    * effort, the call to this method does nothing.
    */
   void registerTerm(Node n, int effort);
+  //-------------------------------------send inferences
+  /** send internal inferences
+   * 
+   * This is called when we have inferred exp => conc, where exp is a set
+   * of equalities and disequalities that hold in the current equality engine.
+   * This method adds equalities and disequalities ~( s = t ) via
+   * sendInference such that both s (resp. t) is either a constant or a term
+   * that already occurs in the equality engine and ~( s = t ) is a consequence
+   * of conc. This function can be seen as a "conservative" version of
+   * sendInference below in that it does not introduce any new non-constant
+   * terms to the state. 
+   * 
+   * The argument c is used for debugging.
```

Comment:
Maybe say something a bit more concrete, e.g. "c is a string identifying the reason for the interference. This string is used for debugging purposes.".